website/docs/en/config/lib/redirect.mdx

```diff
@@ -1,9 +1,23 @@
 # lib.redirect
 
+:::info
+
+Redirect is the specific configuration for bundleless mode (set [lib.bundle](/config/lib/bundle) to `false`). It will not take effect in bundle mode because, all output files are packaged into a single file. As a result, import paths do not exist and do not need to be redirected.
```

Comment:
```suggestion
Redirect is the unique configuration for bundleless mode (set [lib.bundle](/config/lib/bundle) to `false`). It will not take effect in bundle mode where all output files are packaged into a single file, eliminating the need for import path redirection.
```